2022/articles/2022-12-16.pod

```diff
@@ -25,7 +25,7 @@ This helps people get started and use the latest version of the languages.
 
 I<So what if perl also provides its pre-built binaries?>
 
-Because perl itself does not depend on external libraries (expect for glibc on linux, etc.),
```

Comment:
I wonder if both versions could be valid (?)

Reply:
Perhaps so. But even if "expect" is the intention, it also needs "except" additionally in order make sense (to me, anyway).

As it's your article you will know better than I will what you meant. :-)

Reply:
Ahah no it's not mine :) 
Anyway I already merged